dev-php/composer/composer-2.6.5.ebuild

```diff
@@ -99,7 +99,4 @@ src_install() {
 	exeinto "/usr/share/${PN}/bin"
 	doexe "bin/${PN}"
 	dosym "../share/${PN}/bin/${PN}" "/usr/bin/${PN}"
-
-	dodoc CHANGELOG.md README.md doc/*.md
```

Comment:
does the newer composer do not deliver documentation with it? Or why did you remove the documentation packaging?

Reply:
As said in the PR comment: When downloading current version of Composer, those files are missing and causing errors. That's why i've removed them. I've not changed the URL for the download. It's the same as before just with the current version.

Reply:
Ouch! I just realized, that I did not even read your comment. :( 
Sorry about that!

I'll check on this briefy - but I wouldn't consider the missing documentation as a complete show stopper. Of course it would be nice to have "something" around... but I guess everyone, needing information does not use the delivered documentation - I guess everyone prefers online resources for an online tool like compser ;)